SMPost/examples/example-normalized.jsonld

```diff
@@ -24,65 +25,132 @@
     "type": "Property",
     "value": "Twitter"
   },
-  "hasText": {
+  "hasText": [
+  {
     "type": "Property",
-    "value": "This is a tweet"
+    "value": "This is a tweet." 
   },
-  "hasImage": {
+    {
+    "type": "Property",
+    "value": "This is another tweet." 
+  }
+  ],
+  "hasImages": [
+  {
     "type": "Property",
     "value": "https://twt.com/image.png"
   },
-  "hasVideo": {
+  {
+    "type": "Property",
+    "value": "https://twt.com/image2.png"
+  }],
+  "hasVideos": [
+    {
     "type": "Property",
     "value": "https://twt.com/video.mp4"
   },
-  "hasPrivacy": {
+  {
+    "type": "Property",
+    "value": "https://twt.com/video2.mp4"
+  }],
+  "hasPrivacyLevel": {
     "type": "Property",
     "value": "public"
   },
   "location": {
     "type": "GeoProperty",
 	"value": {
-	 "type": "Point",
-	 "coordinates": 
+	"type": "Point",
+	"coordinates": 
 	 [
 	  40.3,
 	  25.5
 	 ]
 	}
   },
-  "hasHashtag": {
+  "hasHashtags": [
+  {
     "type": "Property",
     "value": "[#sample,#tag]"
   },
-  "hasThumbnail": {
+    {
+    "type": "Property",
+    "value": "[#sample2,#tag2]"

```

Comment:
Need quotes here for the values in the array

Reply:
Fixed.